src/Packagist/WebBundle/Resources/public/css/main.css

```diff
@@ -1003,10 +1003,10 @@ ul.packages .metadata-block:last-child {
 .package-aside .details {
   padding-left: 0;
 }
-.package .package-aside i, .package .readme-expander i {
+.package .package-aside i:not(.fa), .package .readme-expander i:not(.fa) {
```

Comment:
Oh, ok I got it (finally).

That's indeed better. But will not works if one time, some people want to use a gliphycon without the gray color.